src/streams/StreamScreen.js

```diff
@@ -61,16 +61,19 @@ class StreamScreen extends PureComponent<Props> {
     return (
       <Screen title="Stream" padding>
         <StreamCard stream={stream} subscription={subscription} />
+        <OptionDivider />
```

Comment:
Quick feedback on commit style:
* These changes are nicely split out into cleanly separate commits -- thanks!
* Please keep the summary (the first line of the commit message) to 60-70 columns. Further details go in paragraphs in the body.
* The prefix "Fix:" doesn't add much -- just leave it out. A good prefix identifies what part of the code is affected; here for this first commit, "StreamScreen:" would be good.

So e.g. this commit message might be
```
StreamScreen: Add spacing between options, for consistency.

This doesn't look great, but it's what we do in other settings screens
in the app. Do the same here for consistency.
```

Reply:
Thanks, I have edited the commit message.